packages/terminal/src/browser/terminal-frontend-module.ts

```diff
@@ -32,18 +34,22 @@ export default new ContainerModule(bind => {
             const child = new Container({ defaultScope: 'Singleton' });
             child.parent = ctx.container;
             const counter = terminalNum++;
-            child.bind(TerminalWidgetOptions).toConstantValue({
-                id: 'terminal-' + counter,
-                caption: 'Terminal ' + counter,
-                label: 'Terminal ' + counter,
+            const domId = options.id || 'terminal-' + counter;
+            const widgetOptions: TerminalWidgetOptions = {
+                title: 'Terminal ' + counter,
```

Comment:
Don't you want to use options.title here, if it's defined?

Reply:
Ahhh, or I guess the `...options` will overwrite it, if `title` is a property of `options`?

Reply:
Yes, correctly.